GTBitsOfGood/canine-assistants
# Handling client-side session errors and loading

## Handle client-side session errors and loading

Issue Number(s): #169 

What does this PR change and why?
PR requirements

* also I think I accidentally branched off my other issue branch so there's some remnants of that, I removed some of them but I think it should be fine once they're both merged

### Checklist


### Critical Changes

- Database change / migration to run
- Environment config change
- Breaking API change

### How To Test

Within SessionManager.jsx, you can manually set status to 'loading' or 'unauthenticated' and attempt to go on protected pages to see either a Loading page or getting kicked off to /login
